src/lib.rs

```diff
@@ -156,6 +156,7 @@ pub enum SanType {
 	Rfc822Name(String),
 	DnsName(String),
 	IpAddress(IpAddr),
+	URI(String),
```

Comment:
Is it possible to move this between `DnsName` and `IpAddress` so that it is consistent in terms of order to RFC 5280? Same goes for all the other places where you've added this.